tests/foreman/cli/test_abrt.py

```diff
@@ -24,7 +24,7 @@ def test_abrt_1(self):
 
         """
 
-    def test_abrt_2(self):
+    def test_positive_create_multiple_reports(self):
```

Comment:
The test case does not have enough information about it.  But you made a right point.  I am going to call these tests `test_positive_create_report` and `test_positive_create_reports` now.